containers/podman-desktop
# Website: updating screenshots for 1.0

### What does this PR do?

Updates the screenshots on podman-desktop.io to show the 1.0 interface which has a different main nav than the existing screenshots.

doesn´t update the following screenshots:
- features / system tray (i don't think it was impacted by the nav changeS)
- home + features / desktop extensions screenshot (i don't have a raw ss for it)

### Screenshot/screencast of this PR

![Screenshot from 2023-05-20 21-45-51](https://github.com/containers/podman-desktop/assets/799683/e0706962-ea09-4757-83ec-a4938f81435f)
![Screenshot from 2023-05-20 21-45-35](https://github.com/containers/podman-desktop/assets/799683/84ed0b1d-c5f0-41d5-bd30-42b1b8607af7)

### What issues does this PR fix or reference?

none but inspired by https://github.com/containers/podman.io/issues/131

### How to test this PR?

Build the website, try in both light and dark modes, note that the following screenshots are new (have the icon based main nav instead of the chunkier older one with text labels):

- Front page - 3 screenshots at top
- Features
  - manage containers 2 screenshots
  - Build, Pull and Push images 2 screenshots
  - Manage Podman resources 2 screenshots
  - Create and start Pods with Podman 2 screenshots
  
  Review screenshots used and make sure they make sense for the topic they're posted in! :-)
